pom.xml

```diff
@@ -237,6 +237,15 @@
     <error-prone.version>2.24.0</error-prone.version>
     <error-prone-support.version>0.14.0</error-prone-support.version>
     <doxia.version>1.12.0</doxia.version>
+    <error-prone.configuration-args>
+      <!-- Disabled rules as we do not see value in them. -->
+      -Xep:JUnitClassModifiers:OFF
+      -Xep:JUnitMethodDeclaration:OFF
+      -Xep:JUnitValueSource:OFF
+      -Xep:LexicographicalAnnotationListing:OFF
+      -Xep:MethodReferenceUsage:OFF
```

Comment:
We shouldn't have many (any?)  violations on this one, since we have an IDEA inspection for this. 

For reference: https://error-prone.picnic.tech/bugpatterns/MethodReferenceUsage/#methodreferenceusage

Reply:
If no violations present now, let's activate.
If there are some let's add it to https://github.com/checkstyle/checkstyle/issues/14137 and we can use comment `until https://github.com/checkstyle/checkstyle/issues/14137`

I highly recommend to merge basic suppression sooner and deal with all activations in separate PRs.

Reply:
The only problem is that this is the only rule in Error Prone Support that is not ready for use yet. For historical reasons this check is already "available" like this, but it's better for now to disable it. We also disabled it internally.

Reply:
For now I added this: `<!-- This check is not finalized. -->`. Is this okay?